### PR TITLE
Removed pug

### PR DESCRIPTION
**Pull request by TylerWasniowski**
We are using react for the frontend now.

### CODE REVIEW

File: packages/backend/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (vikr01) [inline packages/backend/.vscode/settings.json]: Might want to add this into the `*.code-workspace` files and the root level `.vscode/settings.json`.

**Comment by vikr01** [comment]
You should set the target branch to `remove-mongo` since this is continuing from that.

File: packages/backend/.vscode/settings.json (hunk 1 above)
Comment (TylerWasniowski) [reply packages/backend/.vscode/settings.json]: Thank you, did that in homepage branch.

**Comment by TylerWasniowski** [comment]
> You should set the target branch to `remove-mongo` since this is continuing from that.

Done, thanks

**Pull request merged**